src/styles/_catalog.scss

```
@@ -20,3 +17,27 @@
     margin-right: 48px;
   }
 }
+
+@media (min-width:1024px) {
+  .catalog {
+    display: grid;
+    grid-template-columns: auto auto auto auto;
```

here and in other places try to use repeat function
```suggestion
    grid-template-columns: repeat(4, auto);
```